server/routes/helpdesk/validators/uploadValidator.test.ts

```diff
@@ -0,0 +1,69 @@
+import validateUpload from './uploadValidator'
+
+describe('uploadValidator', () => {
+  it(`should validate given no uploaded file`, () => {
+    const upload: Express.Multer.File = null
+
+    const errors = validateUpload(upload)
+
+    expect(errors).toStrictEqual([])
+  })
+
+  it(`should validate given a valid uploaded file`, () => {
+    const upload = {
+      size: 1,
+      mimetype: 'image/jpg',
+    } as Express.Multer.File
+
+    const errors = validateUpload(upload)
+
+    expect(errors).toStrictEqual([])
+  })
+
+  it(`should not validate given an empty uploaded file`, () => {
```

Comment:
Using "validate" and "not validate" confuses me - not validate imlpies that the validation didn't occur, not that the validate failed. So I tend to use "valid" and "invalid" to refer to the validation result.

Reply:
Thats a fair comment - yes, it does 'validate' in all cases, and yes, perhaps 'valid' and 'invalid' are better terms. I'll address that (any my other validators for this story) in my next PR